test/op2ext.test.cpp

```diff
@@ -1,9 +1,7 @@
+#include "../src/StringConversion.h"
```

Comment:
We should configure the `AdditionalIncludeDirectories` setting of the test project so we don't need relative references like this. Ideally the includes (and other code) in the test project should be good example code for client libraries to steal and model proper library usage from.